dockers/base-cuda/Dockerfile

```
@@ -91,18 +93,18 @@ RUN \
     pip config set global.cache-dir false && \
     # set particular PyTorch version \
     for fpath in `ls requirements/**/*.txt`; do \
-      python ./requirements/pytorch/adjust-versions.py $fpath ${PYTORCH_VERSION}; \
+        python ./requirements/pytorch/adjust-versions.py $fpath ${PYTORCH_VERSION} ; \
```

Dockers use 2 spaces

```suggestion
      python ./requirements/pytorch/adjust-versions.py $fpath ${PYTORCH_VERSION} ; \
```